android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -663,12 +673,23 @@ private void closeAllFloatingPanels()
     }
   }
 
-  public void closeMenu(@Nullable Runnable procAfterClose)
+  public boolean closeMenu()
```

Comment:
You don't use this return value, is it really needed?

Reply:
I use `closemenu()` on line 1092 in `onBackPressed` callback https://github.com/organicmaps/organicmaps/pull/2006/files#diff-957993735e10f75c9ccaaf6a794c90089c6e1e217682ea9aedcab85bfd806e36R1092. I don't use `closemenu(@Nullable Runnable procAfterClose)` return value but I added it for consistency.

Reply:
The check is useful to know if calling the function actually closed the menu. This is useful in the `onBackPressed` function to stop when an action has been taken (to make sure a single press does not close everything at once). This is what was done in similar functions so I did the same for consistency.

But as `closemenu(@Nullable Runnable procAfterClose)` is never called in `onBackPressed`, the return value is not useful right now.

Reply:
Let's don't copy old bad practices. If the return value is not needed then remove it. If it is used, then it makes sense at least to add a comment to this function (javadoc style) `// returns false if the menu was already closed.`

Reply:
Fixed in https://github.com/organicmaps/organicmaps/pull/2006/commits/d273ffea50f3f61a3defb0eefe9c5c252c237e08